### PR TITLE
Make the game work [2/2]

### PR DESCRIPTION
**Pull request by Scrabouillmax**
The players can go through the entire game 🎉 

When the game is finished, they are redirected to the main game screen.

### CODE REVIEW

File: server/src/main/java/com/coincoinche/websockets/controllers/GameController.java [hunk 1]
```diff
@@ -5,16 +5,10 @@
...
-import com.coincoinche.events.CardPlayedEvent;
-import com.coincoinche.events.Event;
-import com.coincoinche.events.EventType;
-import com.coincoinche.events.PlayerBadeEvent;
-import com.coincoinche.events.PlayingTurnStartedEvent;
-import com.coincoinche.events.RoundPhaseStartedEvent;
-import com.coincoinche.events.RoundStartedEvent;
-import com.coincoinche.events.TurnStartedEvent;
+import com.coincoinche.engine.teams.Team;
+import com.coincoinche.events.*;
```
Comment (nockty) [inline server/src/main/java/com/coincoinche/websockets/controllers/GameController.java]: We need to replace this by a regular import.

**Review by nockty — APPROVED** [review]
Nice! LGTM.

File: server/src/main/java/com/coincoinche/websockets/controllers/GameController.java (hunk 1 above)
Comment (Scrabouillmax) [reply server/src/main/java/com/coincoinche/websockets/controllers/GameController.java]: IntelliJ does that automatically, I need to change the settings somewhere it's annoying :/

Comment (Scrabouillmax) [reply server/src/main/java/com/coincoinche/websockets/controllers/GameController.java]: Done, I changed my settings

**Pull request merged**